content/docs/user-guide/experiment-management/persisting-experiments.md

```diff
@@ -0,0 +1,99 @@
+# Persisting Experiments
+
+When you run an experiment with `dvc exp run`, it produces temporary objects in
+the Git repository. These are listed with `dvc exp list` and `dvc exp show`.
```

Comment:
We need to put this in the nav sidebar please.

Reply:
Done! Thanks.